test/Faker/Container/ContainerBuilderTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Faker\Test\Container;
+
+use Faker\Container\ContainerBuilder;
+use Faker\English\FileExtension;
+use Faker\Test\TestCase;
+use Psr\Container\ContainerInterface;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+final class ContainerBuilderTest extends TestCase
+{
+    public function testBuildEmpty()
+    {
+        $builder = new ContainerBuilder();
+        $container = $builder->build();
+
+        self::assertInstanceOf(ContainerInterface::class, $container);
+    }
+
+    public function testBuild()
+    {
+        $builder = new ContainerBuilder();
+        $builder->add(FileExtension::class);
+        $container = $builder->build();
+
+        self::assertInstanceOf(ContainerInterface::class, $container);
```

Comment:
What are we testing here?

Reply:
It is just a happy path test that make sure we can actually build the most simple container.